app/assets/sass/application-common.scss

```diff
@@ -23,25 +23,35 @@
 
 // Tasklist page start
 li.task-item {
-    border-top: 1px solid $grey-2;
-    line-height: 1.5;
-    padding: 20px 0 10px 0;
-    position: relative;
-
-    .progress-status-tag {
-      background-color: $black;
-      color: $white;
-      padding: 0 8px;
-      position: absolute;
-      right: 0;
-      top: 0;
-    }
+  border-top: 1px solid $grey-2;
+  line-height: 1.5;
+  padding: 20px 0 10px 0;
+  position: relative;
+
+  .task-item-header {
+    float: left;
+    width: 100%;
+    clear: both;
```

Comment:
I think we could style this more simply by using `overflow: hidden` rather than these 3 styles?